server/index.ts

```diff
@@ -42,6 +48,34 @@ app.post('/api/games', async (request, response) => {
   }
 });
 
+// Fetching from twitch API
+
+app.get('/api/twitchgames', async (_req, res) => {
+  console.log('/twitchgames endpoint called');
+  const options = {
+    'Client-ID': `${process.env.CLIENT_ID}`,
+    Authorization: `Bearer ${process.env.AT}`,
```

Comment:
```suggestion
    Authorization: `Bearer ${process.env.TOKEN}`,
```
Please no abbreviations